miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/ApiClient.kt

```diff
@@ -79,6 +80,17 @@ internal class ApiClient @VisibleForTesting constructor(
         return requestExecutor.executeRequest(request)
     }
 
+    @Throws(MiniAppSdkException::class)
+    suspend fun fetchMiniAppManifest(miniAppId: String, versionId: String): MetadataEntity {
```

Comment:
We can have unit test here.

Reply:
Thanks, I think I have added testcase for this. I'll recheck in next PR.